core/ui.lua

```diff
@@ -323,7 +324,7 @@ function buildAdditionsTab(mod)
 
 	local modNodes = {}
 	table.insert(modNodes, t)
-	return {
+	return (#leftside_nodes > 0 and #rightside_nodes > 0 ) and {
```

Comment:
This doesn't seem right...? It's possible for one column to be empty and the tab should still show

Reply:
It isn't, this should be `or` instead of `and`.